src/App.jsx

```diff
@@ -153,9 +153,68 @@ const Home = () => (
                     </li>
                 </ul>
             </div>
-        </div>        <div className="row">
+        </div>
+        <div className="row">
             <Preview />
         </div>
+        <div className="row">
+            <div className="card col-md-4">
+                <div className="card-body">
+                    <h2 className="card-title">
+                       About
+                    </h2>
+                    <p className="card-text">Stats Interactives are a set of interactives used to illustrate introductory concepts in statistics. The project was built by Prof. Doru Cojoc in collaboration with the Center for Teaching and Learning.</p>
```

Comment:
Stats Interactives `is` a set?

Reply:
This is kind of ambiguous. I used 'are' because 'Interactives' is plural, but 'Stats Interactives' is singular. I'll change it.